elastic/integrations
# [Synthetics] Break out synthetics dashboard integration with adhoc data views

## Proposed commit message



Break out Elastic Synthetics Dashboards integration to avoid bundling it on a default install. Also, removed the static data view in favor ad-hoc.

## Checklist


## How to test this PR locally

1. Build both `Elastic Synthetics` and `Elastic Synthetics Dashboards` locally.
2. Install them remotely to a cloud cluster.
3. Check `Elastic Synthetics` no longer includes dashboard assets.
4. Check `Elastic Synthetics Dashboards`  dashboards are displayed on the list and billing data is reflected.

## Related issues


- Closes #7755.
